### PR TITLE
feat(rhdh-setup): Install RHDH in setup.sh and test catalog listing

### PR DESCRIPTION
**Pull request by pmacik**

### CODE REVIEW

File: ci-scripts/setup.sh [hunk 1]
```diff
@@ -5,3 +5,17 @@ set -o errexit
...
+
+GITHUB_TOKEN=$(cat /usr/local/ci-secrets/backstage-performance/github.token)
+QUAY_TOKEN=$(cat /usr/local/ci-secrets/backstage-performance/quay.token)
+
+
+rm -rvf .rhdh-setup.git*
+
+cd $(mktemp -d .rhdh-setup.git-XXX)
+
+git clone --branch main git@github.com:yogananth-subramanian/rhdh-setup.git .
```
Comment (pmacik) [inline ci-scripts/setup.sh:19]: This will be eventually incorporated into this repo... but for now I just consume @yogananth-subramanian's repo

**Comment by yogananth-subramanian** [comment]
LGTM!

**Review by yogananth-subramanian — APPROVED** [review]
lgtm

**Pull request merged**